README.md

```diff
@@ -63,6 +63,26 @@ curl http://localhost:8080/v1/completions -H "Content-Type: application/json" -d
    }'
 ```
 
+## Helm Chart Installation
```

Comment:
I guess we should update the readme and remove references to the old deployment, this is much better!